hw_futures.dart

```diff
@@ -0,0 +1,64 @@
+void main() async {
+  print("Program started");
+
+  // Task 1: Асинхронне отримання імені
+  // Task 3: Послідовне виконання Future
+
+  final timer = Stopwatch();
+  timer.start();
```

Comment:
You can also write it like this
```suggestion
  final timer = Stopwatch()..start();
```